app/features/jsos/JsosExtractor.ts

```diff
@@ -16,9 +16,46 @@ export class ExtractedCourse {
   ) {}
 }
 
+export class EducationProgram {
+  constructor(
+    readonly name: string,
+    readonly id: string
+  ) {}
+}
```

Comment:
Nie lepiej po prostu tworzyć interface zamiast takiego obiektu? 
```suggestion
IEducationProgram {
  readonly name: string
  readonly id: string
}
```

Reply:
Dlatego, że według mnie to jest  wygodniejsza forma tworzenia pudełka na jakieś dane. Bo mogę włożyć te dane po prostu: new EducationProgram(cos, inne). Ale to kwestia prywatnych preferancji ;)